components/sub-components/NavBar2.jsx

```diff
@@ -0,0 +1,18 @@
+import { FaRegComments } from "react-icons/fa";
+import { RiHome2Line } from "react-icons/ri";
+import { MdOutlineTopic } from "react-icons/md";
+import { Link } from 'react-router-dom';
+
+const NavBar2 = () => {
+
+    
+  return (
+      <nav className="navbar" id="myNavbar">
+        <Link to="/comments"><FaRegComments /></Link>
+        <Link to="/" id="home-button"><RiHome2Line /></Link>
+        <Link to="/topics" id="topics-button"><MdOutlineTopic /></Link>
+      </nav>
+  );
```

Comment:
`NavBar2` seems like a vague name, in northcoders we also previously had components like this in our repo's such as `button1`, `button2`, `button3` etc. 

It's easier for development if you move away from that and move towards names that tell you about what they are, for example this is the nav bar you show on the article page so perhaps `ArticleInfoNavBar`